rust/rope/src/breaks.rs

```diff
@@ -47,7 +47,7 @@ impl Leaf for BreaksLeaf {
     fn push_maybe_split(&mut self, other: &BreaksLeaf, iv: Interval) -> Option<BreaksLeaf> {
         //eprintln!("push_maybe_split {:?} {:?} {}", self, other, iv);
         let (start, end) = iv.start_end();
-        let start_test = if iv.is_start_closed() { start } else { start + 1 };
+        let start_test = start + 1;
```

Comment:
I think this would be better as `start < v` below. Of course, I expect this to change a lot, so doesn't matter much.